src/app/lib/presentation/diagnostic/expanded_diagnostic_page.dart

```diff
@@ -28,37 +31,49 @@ class _ExpandedDiagnosticPageState extends State<ExpandedDiagnosticPage> {
         children: <Widget>[
           Column(
             children: <Widget>[
-              Row(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: <Widget>[
-                  SelectableDiagnosticButton(
-                    label: 'Navigation',
-                    isSelected: _selectedIndex == 0,
-                    onPressed: () {
-                      setState(() {
-                        _selectedIndex = 0;
-                      });
-                    },
-                  ),
-                  SelectableDiagnosticButton(
-                    label: 'Device Info',
-                    isSelected: _selectedIndex == 1,
-                    onPressed: () {
-                      setState(() {
-                        _selectedIndex = 1;
-                      });
-                    },
-                  ),
-                  SelectableDiagnosticButton(
-                    label: 'Environment',
-                    isSelected: _selectedIndex == 2,
-                    onPressed: () {
-                      setState(() {
-                        _selectedIndex = 2;
-                      });
-                    },
-                  ),
-                ],
+              SingleChildScrollView(
```

Comment:
I'm guessing this is equivalent to a TabView? It's probably worth some comments.

Reply:
Also, isn't there an existing TabView that we could reuse?

Reply:
I didn't take time to look for existing flutter component. I'll check and if I don't find any I'll add comments

Reply:
I used `TabBar` instead. It is also cleaner that way